Sources/Commands/SwiftTestTool.swift

```diff
@@ -66,7 +66,7 @@ public class TestToolOptions: ToolOptions {
             return .version
         }
 
-        if shouldRunInParallel {
+        if shouldRunInParallel || numberOfWorkers>0 {
```

Comment:
I think we should add this option in addition to the --parallel option. If we get number of workers without --parallel, we should display an error.

Reply:
Perfect! I'll work on it.

Reply:
Please check the changes I made.
This is my first PR ever, so, feel free to give any any advice. 

Thanks!